packages/client/src/layout/Navigator.re

```diff
@@ -14,7 +14,10 @@ let make = (~route: Route.t, ~loggedInUser: option(Types.user), _children) => {
     | (AgentIndex, _) => AgentIndex'.toEl(loggedInUser)
     | (EntityShow(id), _) => EntityShow'.toEl({id: id}, loggedInUser)
     | (EntityIndex, _) => EntityIndex'.toEl(loggedInUser)
-    | (Profile, _) => Profile'.toEl(loggedInUser)
+    | (Profile, Some(loggedInUser)) =>
+      <FillWithSidebar loggedInUser>
+        <Profile loggedInUser />
+      </FillWithSidebar>
```

Comment:
Why was this needed? Seems a bit hacky.

Reply:
I have to use <ComponentStyle> it is more readable and understandable. If we use functions instead of components we will make the hierarchy of the app too complicated. How it is now for instance with the "authentication" flow.

Reply:
Ok, that works. I'm not sure, will keep in mind.